### PR TITLE
CIS v2 Chart changes

### PR DESCRIPTION
**Pull request by prachidamle**
This is the corresponding charts PR for cis-operator changes https://github.com/rancher/cis-operator/pull/72

https://github.com/rancher/cis-operator/issues/60
https://github.com/rancher/security-scan/issues/28
https://github.com/rancher/security-scan/issues/29
https://github.com/rancher/rancher/issues/30450

### CODE REVIEW

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 image:
   cisoperator:
-    repository: rancher/cis-operator
-    tag: v1.0.3-rc2
+    repository: prachidamle/cis-operator
+    tag: a3
```
Comment (prachidamle) [inline packages/rancher-cis-benchmark/charts/values.yaml:8]: This is a placeholder image tag to be replaced after cis-operator changes are merged

File: packages/rancher-cis-benchmark/charts/templates/scanprofile-rke-1.5-permissive.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 apiVersion: cis.cattle.io/v1
 kind: ClusterScanProfile
 metadata:
-  name: rke-profile-permissive
+  name: rke-profile-permissive-1.5
```
Comment (mrajashree) [inline packages/rancher-cis-benchmark/charts/templates/scanprofile-rke-1.5-permissive.yml:5]: question: when we upgrade from the older version of chart to this, will the ClusterScanProfile CRs named `rke-profile-permissive` and `rke-profile-hardened` get deleted by the upgrade?

Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/templates/scanprofile-rke-1.5-permissive.yml:5]: yes they got deleted by the upgrade

**Review by mrajashree — APPROVED** [review]

**Review by mrajashree — APPROVED** [review]

**Review by cbron — APPROVED** [review]
I don't have context on the scan templates and crds, but other stuff looks good to me.

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 3]
```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.3-rc2
+    tag: v1.0.3-rc3
   securityScan:
     repository: rancher/security-scan
-    tag: v0.2.2-rc1
+    tag: v0.2.2-rc2
```
Comment (cbron) [inline packages/rancher-cis-benchmark/charts/values.yaml:11]: Need to ensure this gets released before 2.5.4

**Pull request merged**